pkg/filters/args.go

```diff
@@ -8,10 +8,17 @@ import (
 )
 
 type ArgFilter struct {
-	Filters map[events.ID]map[string]StringFilter // key to the first map is event id, and to the second map the argument name
+	Filters map[events.ID]map[string]*StringFilter // key to the first map is event id, and to the second map the argument name
 	Enabled bool
 }
 
+func NewArgFilter() *ArgFilter {
+	return &ArgFilter{
+		Filters: map[events.ID]map[string]*StringFilter{},
+		Enabled: false,
```

Comment:
Default boolean value in go is false, so no need to initialize enabled fields in the constructors

Reply:
It was done to be explicit, but I can remove it.